src/components/aside/WorkspaceInfo.vue

```diff
@@ -119,7 +116,7 @@ export default Vue.extend({
         },
         popoverProps:{
           top: '65px',
-          left: '208px',
+          left: '215px',
```

Comment:
still not centered 

![image](https://user-images.githubusercontent.com/15259299/142401476-59c24ebe-8ad8-4b1e-8666-bdc4de7e6fb6.png)

Reply:
is there any way to detect position automatically?

Reply:
@talyguryn I tried to use the event method `getboundingclientrect()` to get the coordinates of the `div` container but the coordinates which I am getting is not accurate.
I also tried Vue `refs` but the problem remains the same.

Reply:
I've improved that logic. now it will be computed automatically